content/en/agent/guide/private-link.md

```diff
@@ -134,7 +134,7 @@ Add `DD_URL: pvtlink.logs.datadoghq.com` in your [Datadog Lambda function][4] en
 {{% /tab %}}
 {{% tab "API" %}}
 
-To send data to Datadog API or consume data from it through this new private link, replace your API call host signature `api.datadoghq.com/api/` with `pvtlink.api.datadoghq.com/api/`.
+To send data to the Datadog API or consume data from it through this new private link, replace your API call host signature `api.datadoghq.com/api/` with `pvtlink.api.datadoghq.com/api/`.
```

Comment:
nit: consistency: here we say 'using this new VPC endpoint', in logs we say 'send logs to the VPC endpoint', but here in the API section we say 'through this new private link',

Reply:
this is also a good point